File: dart/DSA/BigO/O(1).dart
List<int> arr = [3, 4, 5, 6, 7, 8];

int pickArrayValue(List<int> array, int index) {
  return array[index];
}

void main(List<String> args) {
  //This is an O(1), No matter how many elements are in the array
  // Just a single operation is performed
  var val = pickArrayValue(arr, 4);
  print(val);
}

File: golang/DSA/BigO/O(1).go
package o_notation

import "fmt"

func main() {
	str := []string{"Come", "Go", "Hello"}

	//This is an O(1), No matter how many elements art in the array
	// Just a single operation is performed
	res := pickArrayValue(str, 2)
	fmt.Println(res)
}

func pickArrayValue(arr []string, index int) string {
	return arr[index]
}
